src/components/reorderer/js/ModuleLayout.js

```diff
@@ -165,6 +165,13 @@ var fluid_2_0 = fluid_2_0 || {};
                 expander: {
                     func: "{that}.computeLayout"
                 }
+            },
+            getRelativePosition: { // an old-fashioned function member
```

Comment:
It's worth noting here why <code>getRelativePosition</code> is stuck as an "old-fashioned function member" instead of an invoker, and, if possible, what will need to be done in the future to fix this.